### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/deployment/concepts.md`

### PR DESCRIPTION
**Pull request by xzmeng**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 47abc02e26346c60678e8eea723af85de8a8b08a at: https://a19f2068.fastapitiangolo.pages.dev

File: docs/zh/docs/deployment/concepts.md [hunk 1]
```diff
@@ -0,0 +1,323 @@
+# 部署概念
+
+在部署 **FastAPI** 应用程序或实际上任何类型的 Web API 时，您可能会关心几个概念，使用它们您可以找到**最合适的**方法来**部署您的应用程序**。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/concepts.md:3]:
```suggestion
在部署 **FastAPI** 应用程序或任何类型的 Web API 时，有几个概念值得了解，通过掌握这些概念您可以找到**最合适的**方法来**部署您的应用程序**。
```

File: docs/zh/docs/deployment/concepts.md [hunk 2]
```diff
@@ -0,0 +1,323 @@
...
+一些重要的概念是：
+
+* 安全性 - HTTPS
+* 启动时运行
+* 重新启动
+* 复制（运行的进程数）
+* 内存
+* 开始前的先前步骤
+
+我们将了解它们将如何影响**部署**。
```
Comment (socket-socket) [inline docs/zh/docs/deployment/concepts.md:14]:
```suggestion
我们接下来了解它们将如何影响**部署**。
```

File: docs/zh/docs/deployment/concepts.md [hunk 3]
```diff
@@ -0,0 +1,323 @@
...
+
+最后，最终目标是能够以**安全**的方式**为您的 API 客户端**提供服务，**避免中断**，并尽可能高效地使用**计算资源**（ 例如远程服务器/虚拟机）。 🚀
```
Comment (socket-socket) [inline docs/zh/docs/deployment/concepts.md:16]:
```suggestion
我们的最终目标是能够以**安全**的方式**为您的 API 客户端**提供服务，同时要**避免中断**，并且尽可能高效地利用**计算资源**（ 例如服务器CPU资源）。 🚀
```

**Review by socket-socket — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 758204d0808cd360c466de2cd7e57d9d8c1b1e22 at: https://8dd1d228.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thanks for your interest! @xzmeng 

Do you think this is ready to merge now with the suggested changes? Please let me know @socket-socket 🤓

**Comment by socket-socket** [comment]
@alejsdev It's good!

**Review by socket-socket — APPROVED** [review]

File: docs/zh/docs/deployment/concepts.md [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+
+* Kubernetes 中的“Init Container”在应用程序容器之前运行
+* 一个 bash 脚本，运行前面的步骤，然后启动您的应用程序
+     * 您仍然需要一种方法来启动/重新启动 bash 脚本、检测错误等。
+
+!!! tip
+
+     我将在以后的章节中为您提供使用容器执行此操作的更具体示例：[容器中的 FastAPI - Docker](./docker.md){.internal-link target=_blank}。
+
+＃＃ 资源利用率
```
Comment (socket-socket) [inline docs/zh/docs/deployment/concepts.md:289]:
```suggestion
## 资源利用率
```

**Review by socket-socket — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit d1a3b0d6bfd9ba0757c78d265e62df1777e2350f at: https://d0b5d225.fastapitiangolo.pages.dev

**Comment by alejsdev** [comment]
Thank you for the help! @xzmeng 🤓 

And thanks for the review @socket-socket 🚀

**Pull request merged**